lib/presto/client/client.rb

```diff
@@ -40,6 +40,10 @@ def resume_query(next_uri)
       return Query.resume(next_uri, @options)
     end
 
+    def cancel_query_id(query_id)
```

Comment:
Just `kill` is appropriate. (That term is used in Presto UI too.)

Reply:
even though Query's instance method is `Query#cancel`...?

Reply:
👍